pkg/proctree/proctree_feed.go

```diff
@@ -40,9 +40,10 @@ func (pt *ProcessTree) FeedFromFork(feed ForkFeed) error {
 	if feed.ChildTid == 0 || feed.ChildPid == 0 {
 		return errfmt.Errorf("invalid child task")
 	}
-	if feed.ParentTid == 0 || feed.ParentPid == 0 {
-		return errfmt.Errorf("invalid parent task")
-	}
+	// Parent PID or TID might be 0 for init (and docker containers)
```

Comment:
Leftover?

Reply:
No no, I wanted to show explicitly why Im not checking this case.